src/views/teachers/landing/landing.jsx

```diff
@@ -85,7 +85,7 @@ var Landing = injectIntl(React.createClass({
                         <FlexRow className="guides-and-tutorials">
                             <div>
                                 <a href="/go">
-                                    <img src="/svgs/teachers/resources.svg" alt="resources icon" />
+                                    <img src="/svgs/teachers/cards.svg" alt="cards icon" />
```

Comment:
@St19Galla if you want to update the PR with the new image, I'll re-review. Thanks!